### PR TITLE
Add Thai Translation

### PR DESCRIPTION
**Pull request by tomzt**
- Create Folder For Thai 
- Translate two file in to Thai Language

### CODE REVIEW

**Comment by tomzt** [comment]
Hi, Maintainers 
Any suggestions to fix it?

**Comment by JakobLichterfeld** [comment]
Thanks for your change!

**Comment by JakobLichterfeld** [comment]
The build fails due to invalid symbols, so we can not merge this in this state.

File: priv/gettext/th/LC_MESSAGES/default.po
Comment (JakobLichterfeld) [inline priv/gettext/th/LC_MESSAGES/default.po:517]: This needs to be "เพื่อให้แน่ใจว่า...

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]

File: priv/gettext/th/LC_MESSAGES/default.po
Comment (tomzt) [reply priv/gettext/th/LC_MESSAGES/default.po]: completed

**Comment by tomzt** [comment]
@JakobLichterfeld Any suggestions ?

File: priv/gettext/th/LC_MESSAGES/default.po
Comment (JakobLichterfeld) [inline priv/gettext/th/LC_MESSAGES/default.po:416]: This is not correct

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]

**Comment by tomzt** [comment]
@JakobLichterfeld review again, Thanks

**Comment by JakobLichterfeld** [comment]
Perfect, ty!

**Pull request merged**
